apps/chat-e2e/README.md

```diff
@@ -59,11 +59,12 @@ CI report includes screenshots for failed tests.
-| `E2E_WORKERS`  | No       | Number of threads to run e2e tests                                                                                                                 | Any number       |                |
-| `E2E_PASSWORD` | No       | A password for e2e authentification                                                                                                                | Any string       |                |
-| `TMS_URL`      | No       | TMS URL                                                                                                                                            | Any string       |                |
-| `ISSUE_URL`    | No       | Issue URL                                                                                                                                          | Any string       |                |
+| Variable       | Required | Description                                                                                                                                        | Available Values   | Default values |
+| -------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | -------------- |
+| `E2E_HOST`     | No       | The host URL for end-to-end testing.                                                                                                               | Any string         |                |
+| `E2E_USERNAME` | No       | Comma separated list of usernames for e2e authentification. The number of users should be more or equal number of workers set in playwright config | Any string         |                |
+| `E2E_WORKERS`  | No       | Number of threads to run e2e tests                                                                                                                 | Any number         |                |
+| `E2E_PASSWORD` | No       | A password for e2e authentification                                                                                                                | Any string         |                |
+| `TMS_URL`      | No       | TMS URL                                                                                                                                            | Any string         |                |
+| `ISSUE_URL`    | No       | Issue URL                                                                                                                                          | Any string         |                |
+| `STORAGE_TYPE` | No       | Storage type used for e2e tests data                                                                                                               | api,browserStorage | api            |
```

Comment:
Maybe it's possible to read this from already defined variables for chat and use API if not set?

Reply:
agree, fixed